aws/sdk/examples/telephone-game/src/main.rs

```diff
@@ -0,0 +1,439 @@
+use anyhow::Context;
+use aws_sdk_polly::{
+    model::{Engine, OutputFormat, VoiceId},
+};
+use aws_sdk_transcribestreaming::{
+    model::{LanguageCode, Media, MediaFormat},
+};
+use clap::ArgMatches;
+use log::{debug, error, info};
+use rodio::{Decoder, OutputStream, Sink};
+use std::time::Duration;
+use tempdir::TempDir;
+use tokio::{
+    io::{AsyncBufReadExt, AsyncWriteExt, BufReader},
+    runtime::Runtime,
+};
+
+/// While playing the telephone game, the user can pass an arg that defines how many times to pass the message through Polly and Transcribe.
+/// This is the default number of iterations to do when the user doesn't specify.
+const DEFAULT_NUMBER_OF_ITERATIONS: u32 = 5;
+/// When running a job/task that takes some time to complete (speech synthesis and transcription), this sets a maximum wait time in seconds before giving up.
+const TASK_TIMEOUT_IN_SECONDS: i32 = 30;
+/// How often to poll for job/task status
+const TASK_WAIT_INTERVAL_IN_SECONDS: i32 = 2;
+
+fn main() {
+    // Read ENV vars from a .env file if one exists
+    // To see logs set RUST_LOG="error,telephone_game=debug"
+    dotenv::dotenv().expect("failed to read env vars");
+    // Start up the logger
+    env_logger::init();
```

Comment:
we have been suggestion people use `tracing_subscriber` in most of the other examples

Reply:
I'll use that